modules/drivers/oracle/test/metabase/driver/oracle_test.clj

```diff
@@ -88,63 +89,63 @@
-                    {:name "sid"}
-                    {:name "service-name"}
-                    {:name "user"}
-                    {:name "password"}
-                    {:name "cloud-ip-address-info"}
-                    {:name "ssl"}
-                    {:name "ssl-use-keystore"}
-                    {:name       "ssl-keystore-options"
-                     :type       "select"
-                     :options    [{:name  "Local file path"
-                                   :value "local"}
-                                  {:name  "Uploaded file path"
-                                   :value "uploaded"}]
-                     :visible-if {:ssl-use-keystore true}}
-                    {:name       "ssl-keystore-value"
-                     :type       "textFile"
-                     :visible-if {:ssl-keystore-options "uploaded"}}
-                    {:name       "ssl-keystore-path"
-                     :type       "string"
-                     :visible-if {:ssl-keystore-options "local"}}
-                    {:name "ssl-keystore-password-value"
-                     :type "password"}
-                    {:name "ssl-use-truststore"}
-                    {:name       "ssl-truststore-options"
-                     :type       "select"
-                     :options    [{:name  "Local file path"
-                                   :value "local"}
-                                  {:name  "Uploaded file path"
-                                   :value "uploaded"}]
-                     :visible-if {:ssl-use-truststore true}}
-                    {:name       "ssl-truststore-value"
-                     :type       "textFile"
-                     :visible-if {:ssl-truststore-options "uploaded"}}
-                    {:name       "ssl-truststore-path"
-                     :type       "string"
-                     :visible-if {:ssl-truststore-options "local"}}
-                    {:name "ssl-truststore-password-value"
-                     :type "password"}
-                    {:name "tunnel-enabled"}
-                    {:name "tunnel-host"}
-                    {:name "tunnel-port"}
-                    {:name "tunnel-user"}
-                    {:name "tunnel-auth-option"}
-                    {:name "tunnel-pass"}
-                    {:name "tunnel-private-key"}
-                    {:name "tunnel-private-key-passphrase"}
-                    {:name "advanced-options"}
-                    {:name "auto_run_queries"}
-                    {:name "let-user-control-scheduling"}
-                    {:name "schedules.metadata_sync"}
-                    {:name "schedules.cache_field_values"}
-                    {:name "refingerprint"}]
-          actual   (->> (driver/connection-properties :oracle)
-                        (driver.u/connection-props-server->client :oracle))]
-      (is (= expected (mt/select-keys-sequentially expected actual))))))
+    (with-redefs [premium-features/is-hosted? (constantly false)]
```

Comment:
added the `with-redefs` call so that `cloud-ip-address-info` is deterministically not included in connection properties in this test (since it's not really relevant to oracle in particular)